cypress/e2e/gui-tests/engageSphere.cy.js

```diff
@@ -0,0 +1,430 @@
+
+    cy.get('table tbody tr').should('have.length', 5);
+
+    cy.get('tr:contains(1)')
+      .find('td:contains(99)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(35)')
+      .find('td:contains(71)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(38)')
+      .find('td:contains(70)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(22)')
+      .find('td:contains(68)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(32)')
+      .find('td:contains(57)')
+      .parent()
+      .children()
+      .first()
+  })
+
+  it('Validates sizes in a table column are in ascending order', () => {
+    cy.intercept(
+      'GET',
+      `${Cypress.env('API_URL')}/customers?page=1&limit=10&size=All`,
+      { fixture: 'allSizeCustomers' }
+    ).as('getAllSizeCustomers')
+
+    cy.get('[data-testid="filter"]').select('All')
+
+    cy.wait('@getAllSizeCustomers')
+
+    cy.get('tr:contains(Number of employees)').children().eq(2).click()
+
+    cy.get('tr:contains(1)')
+      .find('td:contains(Small)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(2)')
+      .find('td:contains(Medium)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(4)')
+      .find('td:contains(Enterprise)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(6)')
+      .find('td:contains(Large Enterprise)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(8)')
+      .find('td:contains(Very Large Enterprise)')
+      .parent()
+      .children()
+      .first()
+  });
+
+  it('Sorts by Size in descending order by default', () => {
+    cy.intercept(
+      'GET',
+      `${Cypress.env('API_URL')}/customers?page=1&limit=10&size=All`,
+      { fixture: 'allSizeCustomers' }
+    ).as('getAllSizeCustomers')
+
+    cy.get('[data-testid="filter"]').select('All')
+
+    cy.wait('@getAllSizeCustomers')
+
+    cy.get('tr:contains(Size)').children().eq(3).click()
+    cy.get('tr:contains(Size)').children().eq(3).click()
+
+    cy.get('tr:contains(8)')
+      .find('td:contains(Very Large Enterprise)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(6)')
+      .find('td:contains(Large Enterprise)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(4)')
+      .find('td:contains(Enterprise)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(2)')
+      .find('td:contains(Medium)')
+      .parent()
+      .children()
+      .first()
+
+    cy.get('tr:contains(1)')
+      .find('td:contains(Small)')
+      .parent()
+      .children()
+      .first()
+  });
+
+  it('Goes back to the customers list when clicking the "Back" button', () => {
+    cy.get('tbody tr').first().click()
+
+    cy.get('tbody tr').should('not.exist')
```

Comment:
Aqui você poderia verificar que a tabela inteira não existe.

```suggestion
    cy.get('table').should('not.exist')
```